.storybook/stories/header/header.stories.ts

```diff
@@ -24,6 +24,18 @@ const defaultStory: Story = args => ({
           <a href="javascript://" class="nav-link nav-text">Home</a>
           <a href="javascript://" class="nav-link nav-text">About</a>
           <a href="javascript://" class="nav-link nav-text">Services</a>
+          <a href="javascript://" class="nav-link nav-text">Services</a>
```

Comment:
Added these for test purposes for the preview link, I might delete them before merging

Reply:
This is the reason I am not accepting the visual changes